docs/PowerAuth-SDK-for-iOS.md

```diff
@@ -1116,6 +1116,31 @@ context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason
 }
 ```
 
+Be aware that the example above doesn't handle all quirks related to the PowerAuth protocol, so you should prefer to use `authenticateUsingBiometry()` function instead:
+
+```swift
+let context = LAContext()
+context.localizedReason = "Please authenticate with biometry"
+PowerAuthSDK.sharedInstance().authenticateUsingBiometry(withContext: context) { authentication, error in
+    guard let authentication = authentication else {
+        let nsError = error! as NSError
+        if nsError.domain == PowerAuthErrorDomain {
```

Comment:
I would add casting to the NSError as part of the if (`if let`). I know it cannot happen since authentication is nil in this case, but someone without the knowledge of the API doesn't know that.